frame/contracts/src/benchmarking/mod.rs

```diff
@@ -808,6 +808,46 @@ benchmarks! {
 		let origin = RawOrigin::Signed(instance.caller.clone());
 	}: call(origin, instance.addr, 0u32.into(), Weight::MAX, None, vec![])
 
+	seal_set_code_hash {
```

Comment:
This is needed to prevent spamming of the output file because we create a lot of unique storage items.

```suggestion
    #[skip_meta]
	seal_set_code_hash {
```